test/parallel/test-fs-readfile.js

```diff
@@ -52,6 +52,23 @@ for (const e of fileInfo) {
     assert.deepStrictEqual(buf, e.contents);
   }));
 }
+// Test readFile size too large
+{
+  const kIoMaxLength = 2 ** 31 - 1;
+
+  const file = path.join(tmpdir.path, `${prefix}-too-large.txt`);
+  fs.writeFileSync(file, Buffer.from('0'));
+  fs.truncateSync(file, kIoMaxLength + 1);
+
+  fs.readFile(file, common.mustCall((err) => {
+    assert.strictEqual(err.name, 'RangeError');
+    assert.strictEqual(err.code, 'ERR_FS_FILE_TOO_LARGE');
+  }));
```

Comment:
Someone may suggest `common.expectsError()` for this, but I find this more readable to be honest. (I'm fine with it either way.)

Reply:
OK. Let's follow the convention